pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+                selections = ["NAME:Selections", "Selections:=", objects, "NewPartsModelFlag:=", "Model"]
+        self.oeditor.Split(
+            selections,
+            [
+                "NAME:SplitToParameters",
+                "SplitPlane:=",
+                planes,
+                "WhichSide:=",
+                sides,
+                "ToolType:=",
+                tool_type,
+                "ToolEntityID:=",
+                tool_entity_id,
+                "SplitCrossingObjectsOnly:=",
+                split_crossing_objs,
+                "DeleteInvalidObjects:=",
+                delete_invalid_objs,
+            ],
+        )
+        self.refresh_all_ids()
+        return [objects] + [i for i in self.object_names if i not in all_objs]
+
+    @pyaedt_function_handler()  # TODO: Deprecate this and add duplicate as an option in the mirror method.
+    def duplicate_and_mirror(
+        self,
+        objid,
+        position,
+        vector,
+        is_3d_comp=False,
+        duplicate_assignment=True,
+    ):
+        """Duplicate and mirror a selection.
+
+        Parameters
+        ----------
+        objid : str, int, or  Object3d
+            Name or ID of the object.
+        position : float
+            List of the ``[x, y, z]`` coordinates or
+            Application.Position object for the selection.
+        vector : float
+            List of the ``[x1, y1, z1]`` coordinates or
+            Application.Position object for the vector.
+        is_3d_comp : bool, optional
+            If ``True``, the method will try to return the duplicated list of 3dcomponents. The default is ``False``.
+        duplicate_assignment : bool, optional
+            If True, the method duplicates selection assignments. The default value is ``True``.
+
+        Returns
+        -------
+        list
+            List of objects created or an empty list.
+
+        References
+        ----------
+
+        >>> oEditor.DuplicateMirror
+        """
+        return self.mirror(
+            objid, position, vector, duplicate=True, is_3d_comp=is_3d_comp, duplicate_assignment=duplicate_assignment
+        )
+        # selections = self.convert_to_selections(objid)
+
+    @pyaedt_function_handler()
+    def mirror(self, objid, position, vector, duplicate=False, is_3d_comp=False, duplicate_assignment=True):
+        """Mirror a selection.
+
+        Parameters
+        ----------
+        objid : str, int, or Object3d
+            Name or ID of the object.
+        position : int or float
+            List of the ``[x, y, z]`` coordinates or the
+            ``Application.Position`` object for the selection.
+        duplicate : bool, optional
+            Whether if duplicate the object before mirror or not. Default is ``False``.
+        is_3d_comp : bool, optional
+            Whether the component is 3D. The default is ``False``. If ``True``, the method
+            tries to return the duplicated list of 3D components.
```

Comment:
```suggestion
            Whether the method is to try to return the duplicated list of 3D components.
            The default is ``False``.
```
This edit assumes the description should be the same as for the earlier ``is_3d_comp`` parameter.